src/xsar/sentinel1_meta.py

```diff
@@ -1066,28 +1066,41 @@ def bursts(self, only_valid_location=True):
             'geometry' is the polygon
 
         """
-        burst_list = self._bursts
-        if burst_list['burst'].size == 0:
-            blocks = gpd.GeoDataFrame()
-        else:
-            bursts = []
-            bursts_az_inds = {}
-            inds_burst, geoloc_azitime, geoloc_iburst, geoloc_line = self._get_indices_bursts()
-            for burst_ind, uu in enumerate(np.unique(inds_burst)):
-                if only_valid_location:
-                    extent = np.copy(burst_list['valid_location'].values[burst_ind, :])
-                    area = box(extent[0], extent[1], extent[2], extent[3])
-
+        if self.multidataset:
+            blocks = None
+            for ss,subswath in enumerate(self.subdatasets):
+                metasub = Sentinel1Meta(subswath)
+                new_block = metasub.bursts()
+                new_index = new_block.index
+                new_index = zip(new_index,np.ones((len(new_block)))*ss)
+                new_index = pd.MultiIndex.from_tuples(new_index, names=["burst", "subswath"])
```

Comment:
index should be a multiindex `('subwath', 'burst')` in this order.  `burst` should be taken from subdataset `metasub.bursts().index`, and `subswath` should be taken from `metasub.dsid` (ie 'IW1', 'IW2', ...)

Reply:
i.e. something like:


|  |   |       geometry |
|--|--|--|
| subswath | burst | |                                                   
| IW1  |     0   |   POLYGON ((1474.000 92.000, 1474.000 20540.000,... |
|  | 1  |    POLYGON ((2963.000 92.000, 2963.000 20540.000,... |
|  |...|...|
| IW2  |     0   |   POLYGON ((1474.000 92.000, 1474.000 20540.000,... |
|  | 1  |    POLYGON ((2963.000 92.000, 2963.000 20540.000,... |
|  |...|...|

Reply:
fixed with https://github.com/umr-lops/xsar/pull/79/commits/b1d3efef23037975ff5f2bee6c697e18c7861971

Reply:
![image](https://user-images.githubusercontent.com/14925067/162725276-13b75736-2ba7-4d36-8424-4224d9672c42.png)